en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -158,33 +169,31 @@ Let's design the REST API as follows:
 
     3. Click **Save**.
 
-6. To build the JSON payload to be sent as the response and then send the response, follow this procedure:
+9. To build the JSON payload with data of the total cases per million in the population, add a variable.
 
-    1. To build the JSON payload with data of the total cases per million in the population, add a variable.
+    Click the last **+** icon in the low-code diagram and click **Variable**. Then enter information as follows:
 
-        Click the last **+** icon in the low-code diagram and click **Variable**. Then enter information as follows:
+    | **Field**         | **Value**                     |
+    |-------------------|-------------------------------|
+    | **Variable Type** | `json`                        |
+    | **Variable Name** | `payload`                     |
+    | Expression        | `{country : country, totalCasesPerMillion : totalCasesPerMillion}` |
 
-         | **Field**         | **Value**                     |
-         |-------------------|-------------------------------|
-         | **Variable Type** | `json`                        |
-         | **Variable Name** | `payload`                     |
-         | Expression        | `{country : country, totalCasesPerMillion : totalCasesPerMillion}` |
+    Click **Save**.
 
-    2. Click **Save**.
+10. To respond with the JSON payload, add a `Return` statement.
 
-    3. To respond with the JSON payload, add a `Return` statement.
+    Click the last **+** icon in the low-code diagram and click **Return**.
 
-        Click the last **+** icon in the low-code diagram and click **Return**.
+    In the **Return Expression** field, enter `payload`.
 
-        In the **Return Expression** field, enter `payload`.
-
-        Click **Save**.
+    Click **Save**.
 
 Now you have completed designing the `CovidStatus` service.
```

Comment:
`CovidStatus` service is not correct, right? Should be "`Statistics` REST API" isn't it?

Reply:
Corrected.